xs/src/libslic3r/PrintConfig.cpp

```diff
@@ -1281,6 +1281,15 @@ PrintConfigDef::PrintConfigDef()
     def->enum_labels.push_back("auto");
     def->default_value = new ConfigOptionFloatOrPercent(15, false);
 
+    def = this->add("small_perimeter_length", coFloat);
+    def->label = __TRANS("Small perimeter length");
+    def->category = __TRANS("Layers and Perimeters");
+    def->tooltip = __TRANS("This setting controls the lenghtof the small perimeters");
+    def->sidetext = "mm";
+    def->cli = "small-perimeter-length=f";
+    def->min = 0;
+    def->default_value = new ConfigOptionFloat(6.5*2*PI);
```

Comment:
Go ahead and leave this a 6.5 and change the usage in xs/src/libslic3r/GCode.cpp to 
`scale_(this->config.get_abs_value("small_perimeter_length")) * 2 * PI `